temporal-opentracing/src/main/java/io/temporal/opentracing/StartSpanContext.java

```diff
@@ -0,0 +1,76 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.opentracing;
+
+public class StartSpanContext {
+
+  private OpenTracingOptions options;
+  private SpanOperationType spanOperationType;
+  private String typeName;
+  private String workflowId;
+  private String runId;
+  private String parentWorkflowId;
+  private String parentRunId;
+
+  public StartSpanContext(
```

Comment:
Maybe I'm not getting something, but why is it called Start*? It's not used to start anything and it's not only for Start operations (There is no RunSpanContext or smth)

Should it be called `SpanNamingContext` with javadoc saying something like:
"Contains all pieces of information needed by OperationNameAndTagsProvider"

Reply:
I called it that because it's used when we start a span. But I see the confusion. I don't like `SpanNamingContext` because it's not just about naming, but tagging.  I'm thinking `SpanCreationContext`

Reply:
Yeah, I think SpanCreationContext is a better name than StartSpanContext

Reply:
Done